### PR TITLE
Fixed alignment of bottom footer

### PR DESCRIPTION
**Pull request by hstandaert**
### Before
screenshot 2019-02-28 at 12 55 36

### After
screenshot 2019-02-28 at 12 55 46

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [x] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

File: styleguide/components/31-molecules/social-list/_social-list.scss [hunk 1]
```diff
@@ -1,18 +1,19 @@
 $social-media-link-width: 1.6rem;
 
 .social-list {
-  width: 100%;
+  display: inline-block;
+  width: auto;
+  max-width: 100%;
```
Comment (delrueba) [inline styleguide/components/31-molecules/social-list/_social-list.scss]: Why is this needed?
Seems silly, but I'm sure you had your reason.

Comment (hstandaert) [reply styleguide/components/31-molecules/social-list/_social-list.scss]: Totally unnecessary; removed it 👍

**Review by delrueba — APPROVED** [review]
